module1/homework/grayscaleImages/test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <utility>  // for std::pair<>
 #include <vector>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
vector is in the header so it's not needed here.

Reply:
+1